web/src/providers/Environment/hooks/useEnvironmentCrud.ts

```diff
@@ -0,0 +1,46 @@
+import {useCallback} from 'react';
+import {message} from 'antd';
+import {
+  useCreateEnvironmentMutation,
+  useDeleteEnvironmentMutation,
+  useUpdateEnvironmentMutation,
+} from 'redux/apis/TraceTest.api';
+import {useConfirmationModal} from 'providers/ConfirmationModal/ConfirmationModal.provider';
+import Environment from 'models/Environment.model';
+
+const useEnvironmentCrud = () => {
+  const [deleteEnvironment] = useDeleteEnvironmentMutation();
+  const [createEnvironment, {isLoading: isCreateLoading}] = useCreateEnvironmentMutation();
+  const [updateEnvironment, {isLoading: isUpdateLoading}] = useUpdateEnvironmentMutation();
+  const {onOpen} = useConfirmationModal();
+
+  const remove = useCallback(
+    (id: string) => {
+      onOpen({
+        title: `Are you sure you want to delete the environment?`,
+        onConfirm: () => deleteEnvironment({environmentId: id}),
+      });
+    },
+    [deleteEnvironment, onOpen]
+  );
+
+  const edit = useCallback(
+    async (environmentId: string, environment: Environment) => {
+      await updateEnvironment({environmentId, environment});
+      message.success('Environment updated successfully');
```

Comment:
We should try to use an standard component for those confirmation messages. I think we have a different one for DataStores. Not a blocker we can handle that in a different task.

Reply:
agree with @jorgeepc comment

Reply:
I believe we added the messages to provide feedback for some of the FE actions. 

I created this issue a long time ago while thinking about adding a standardized notification system https://github.com/kubeshop/tracetest/issues/1478

Let's all think about that and how we can introduce it in more general way 😄